metaflow/plugins/kfp/kfp.py

```diff
@@ -82,16 +65,42 @@ class KfpComponent(object):
     def __init__(
         self,
         name: str,
-        cmd_template: str,
+        init_cmd: str,
+        cd_into_metaflow_package_cmd: str,
+        clean_volume_cmd: str,
+        task_id: str,
+        task_id_template: str,
+        step_name: str,
+        flow_name: str,
         total_retries: int,
+        namespace: str,
+        tags: List[str],
+        need_split_index: bool,
+        environment_type: str,
+        logger_type: str,
+        monitor_type: str,
+        user_code_retries: int,
```

Comment:
these variables are no longer KFP component variables, but are Metaflow bootstrap variables.
Could you please organize or wrap them as such?  Possibly a new `dataclass` for these variables.

Reply:
Created a new `dataclass` called `MetaflowBootstrapVars`.